include/hal/library/responder/measlib.h

```diff
@@ -149,6 +149,23 @@ extern bool libspdm_generate_measurement_summary_hash(
     uint8_t measurement_summary_hash_type,
     uint8_t *measurement_summary_hash,
     uint32_t measurement_summary_hash_size);
+
```

Comment:
MEL is its own capability. This should be within `LIBSPDM_ENABLE_CAPABILITY_MEL_CAP`.

Reply:
Thanks. I have added the `LIBSPDM_ENABLE_CAPABILITY_MEL_CAP` and moved APIs under the MACRO.

And in current design, the APIs exist only when both `LIBSPDM_ENABLE_CAPABILITY_MEAS_CAP` and `LIBSPDM_ENABLE_CAPABILITY_MEL_CAP` are enabled.

Because the MEL feature need measurement feature firstly.  Do you agree this design?

Reply:
The `MEL_CAP` / `MEAS_CAP` capabilities can be completely separate. No need for libspdm to tie them together.

Reply:
Thanks. I have split these two MACRO.